### ISSUE TITLE
`@unidriver/protractor` - has not `dist` folder

### ISSUE THREAD

**Issue by hepiyellow** [wix-incubator/unidriver#60]
`@unidriver/protractor@1.1.0` has no `dist` folder.
but `package.json` has `"main": "dist/index.js"`.

Probably because of this:
```
"scripts": {
    "build": "echo 'TBD'",
    "test": "echo 'TBD'"
  },
```

**Comment by hepiyellow** [comment]
@GabiGrin @borisd9 FYI, Can you fix this today?

**Comment by GabiGrin** [comment]
will be fixed with #63

**Comment by GabiGrin** [comment]
@hepiyellow pushed and released, please confirm
